src/backend/distributed/shared_library_init.c

```diff
@@ -1119,6 +1122,18 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomStringVariable(
+		"citus.set_local_whitelist",
+		gettext_noop("Sets whitelist of GUCs permitted in SET LOCAL."),
+		NULL,
+		&SetLocalWhitelistString,
+		"",
+		PGC_POSTMASTER,
```

Comment:
Can we make this PGC_USERSET? Otherwise it isn't very useful in cloud.

Reply:
Sure; did it (pushing soon).